contracts/adapters/Ragequit.sol

```diff
@@ -67,6 +67,12 @@ contract RagequitContract is IRagequit, DaoConstants, MemberGuard {
         uint256 lootToBurn,
         address[] memory tokens
     ) external override {
+        // Checks if the are enough shares and/or loot to burn
+        require(sharesToBurn + lootToBurn > 0, "insufficient shares/loot");
+
+        // Checks if the member is not in jail due to a Guild Kick
```

Comment:
@adridadou I haven't changed this yet, but I thinking about checking if the member is not in jail before processing the ragequit. Mainly because the member might be part of a Guild Kick proposal.

Reply:
yes it makes sense. This way we make sure we don't double process the member

Reply:
Done.